### PR TITLE
[tests-only][full-ci] extend test for trying to move file/folder within shares-personal drive with secure view role

### PR DESCRIPTION
**Pull request by anon-pradip**
## Description

This PR updates the previous scenarios regarding sharee trying to move file/folder within the shared folder with permission secure view too.
Scenario Updated:
```feature
Scenario Outline: sharer renames the shared item (spaces webdav)
Scenario Outline: share receiver renames the shared item (old/new webdav)
Scenario Outline: share receiver renames the shared item (spaces webdav)
Scenario: receiver tries to rename a received share with read permissions inside the Shares folder
```

## Related Issue

- Part of: https://github.com/owncloud/ocis/issues/9334

## How Has This Been Tested?

- test environment:
- Locally
- CI
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: tests/acceptance/features/coreApiShareManagementToShares/moveReceivedShare.feature [hunk 1]
```diff
@@ -201,15 +208,16 @@ Feature: sharing
     But as "Alice" file "/folderToShare/fileInside" should not exist
 
 
-  Scenario: receiver tries to rename a received share with read permissions inside the Shares folder
```
Comment (S-Panta) [inline tests/acceptance/features/coreApiShareManagementToShares/moveReceivedShare.feature:204]: read permission?

File: tests/acceptance/features/coreApiShareManagementToShares/moveReceivedShare.feature [hunk 2]
```diff
@@ -62,7 +62,7 @@ Feature: sharing
       | space           | Personal      |
       | sharee          | Carol         |
       | shareType       | user          |
-      | permissionsRole | Viewer        |
+      | permissionsRole | <permissions-role>        |
```
Comment (saw-jan) [inline tests/acceptance/features/coreApiShareManagementToShares/moveReceivedShare.feature:65]: this should not be necessary

**Review by saw-jan — CHANGES_REQUESTED** [review]

File: tests/acceptance/features/coreApiShareManagementToShares/moveReceivedShare.feature (hunk 2 above)
Comment (anon-pradip) [reply tests/acceptance/features/coreApiShareManagementToShares/moveReceivedShare.feature:65]: May I please get some clarification(s) why wouldn't it be necessary?

File: tests/acceptance/features/coreApiShareManagementToShares/moveReceivedShare.feature (hunk 1 above)
Comment (anon-pradip) [reply tests/acceptance/features/coreApiShareManagementToShares/moveReceivedShare.feature:204]: It was already written by someone else. Will not it be better to keep it like:
`receiver tries to rename a received share inside the Shares folder`

Comment (anon-pradip) [reply tests/acceptance/features/coreApiShareManagementToShares/moveReceivedShare.feature:204]: > read permission?

"read permissions" means except "Can edit" permission

File: tests/acceptance/features/coreApiShareManagementToShares/moveReceivedShare.feature (hunk 2 above)
Comment (saw-jan) [reply tests/acceptance/features/coreApiShareManagementToShares/moveReceivedShare.feature:65]: share role should not matter for sharer to move/rename the file.

File: tests/acceptance/features/coreApiShareManagementToShares/moveReceivedShare.feature [hunk 3]
```diff
@@ -86,22 +86,23 @@ Feature: sharing
       | /remote.php/dav/spaces/%shares_drive_id% | /remote.php/dav/spaces/%spaceid% |
       | /dav/spaces/%shares_drive_id%            | /remote.php/dav/spaces/%spaceid% |
 
+
   @issue-8242
```
Comment (saw-jan) [inline tests/acceptance/features/coreApiShareManagementToShares/moveReceivedShare.feature:90]:
```suggestion

  @issue-8242
```

**Review by saw-jan — CHANGES_REQUESTED** [review]

File: tests/acceptance/features/coreApiShareManagementToShares/moveReceivedShare.feature [hunk 4]
```diff
@@ -85,23 +85,23 @@ Feature: sharing
       | dav-path                                 | dav-path-personal                |
       | /remote.php/dav/spaces/%shares_drive_id% | /remote.php/dav/spaces/%spaceid% |
       | /dav/spaces/%shares_drive_id%            | /remote.php/dav/spaces/%spaceid% |
-
+
```
Comment (saw-jan) [inline tests/acceptance/features/coreApiShareManagementToShares/moveReceivedShare.feature:88]:
```suggestion

```

**Pull request merged**